lib/photos.py

```diff
@@ -45,6 +45,7 @@ def __call__(self, driver):
 	chrome_options.add_argument('--no-sandbox')
```

Comment:
Same case as my previous comment. This should be deleted. Otherwise, when using WSL or Windows, chrome will end up with `--no-sandbox --no-sandbox` as arguments.

Reply:
This must be an old version of the PR; it was already moved to in a [previous commit](https://github.com/mxrch/GHunt/pull/20/commits/bcb988776188a31670a89de113f0f66e5e4c0839#diff-79b9c26160b3ad84b0622b2a7cee63c6R54).